packages/utils/src/libp2p/dialHelper.ts

```diff
@@ -136,15 +150,22 @@ async function queryDHT(
   }
 
   // Libp2p's return types tend to change every now and then
-  if (dhtResponse == null) {
+  if (relayers.length == 0) {
     return { status: DialStatus.DHT_ERROR, query: destination }
   }
 
+  log(`found ${relayers.map((relay) => relay.id.toB58String()).join(` ,`)} for node ${destination.toB58String()}.`)
```

Comment:
Normal string can be used in `join` arg, there's no template in it.

Reply:
changed